sql/src/main/java/org/apache/druid/sql/calcite/view/DruidViewMacro.java

```diff
@@ -51,10 +52,11 @@ public DruidViewMacro(final PlannerFactory plannerFactory, final Escalator escal
   public TranslatableTable apply(final List<Object> arguments)
   {
     final RelDataType rowType;
-    try (final DruidPlanner planner = plannerFactory.createPlanner(null)) {
+    final AuthenticationResult authenticationResult = escalator.createEscalatedAuthenticationResult();
+    try (final DruidPlanner planner = plannerFactory.createPlanner(null, authenticationResult)) {
       // Using an escalator here is a hack, but it's currently needed to get the row type. Ideally, some
```

Comment:
This comment should move up a couple lines, to where the escalator is actually used.

Reply:
👌